docs/introduction/exchanges.md

```diff
@@ -1,10 +1,46 @@
-# Exchange
+# Exchange and Transfer
 
 This vocabulary subset looks at exchanges of resources from an independent or neutral viewpoint (not the viewpoint of one of the Agents in the exchange). For example, from one Agent's viewpoint, the exchange may be a Purchase, from the other Agent's viewpoint, it might be a Sale. From the neutral viewpoint, it is an exchange of resources, with usually at least two flows of resources, one from each direction. So for example, the seller might give some goods to the buyer, and the buyer might give some money to the seller. Or in a barter exchange, one agent might give the other some books, and the other agent might compensate with some cookies.
 
 This differs from (for example) the [Good Relations Conceptual Model](http://wiki.goodrelations-vocabulary.org/Documentation/Conceptual_model), which we like and want to intertrade with. But Good Relations assumes Compensation in the form of money; the compensation itself is not a separate promise; and the actual flow events are not part of the model. (That's not a criticism. Good Relations has different goals, and a more minimal model makes sense for them.)
 
 In ValueFlows, we want to track not only the offers and promises, but also the actual flows of resources in networks, in all directions. And we support exchanges that don't involve money as well as those that do.
 
-We also support non-reciprocal one-way transfers, such as in a gift economy.
+We also support non-reciprocal one-way transfers, such as in a gift economy.  An exchange implies at least two transfers.
+
+### Transfers
+
+We want to emphasize that although ownership is the primary right transferred in the current economic system (along with renting or leasing), we seek to focus on transfers not necessarily governed by the legal context of property ownership.  This puts the focus on economic coordination.
```

Comment:
If Transfer supposed to handle transferring various kinds of rights and responsibilities, where do you specify what kind of them gets transferred?